doc/src/cylc-user-guide/cug.tex

```diff
@@ -3779,7 +3794,7 @@ \subsubsection{Trigger Types}
 message. Other tasks can trigger off the externally triggered task as required,
 of course.
 
-\lstinline=$CYLC_DIR/examples/satellite/ext-triggers/suite.rc= is a working
+\lstinline=<cylc-dir>/etc/examples/satellite/ext-triggers/suite.rc= is a working
```

Comment:
I like the `<substitution>` syntax, in the rose documentation we now have a pygments lexer for highlighting these in code blocks e.g: http://metomi.github.io/rose/doc/html/cheat-sheet.html

Reply:
Nice, looking foward to going cylc docs going this way...